src/platform/test/plugin_functional/test_suites/core_plugins/rendering.ts

```diff
@@ -203,7 +203,8 @@ export default function ({ getService }: PluginFunctionalProviderContext) {
         'vis_type_vislib.readOnly (boolean?|never)',
         'vis_type_xy.readOnly (boolean?|never)',
         'vis_type_vega.enableExternalUrls (boolean?)',
-        'xpack.actions.email.domain_allowlist (array)',
+        'xpack.actions.email.domain_allowlist (array?)',
```

Comment:
Question: Would values contained in the domain_allowlist or the recipient_allowlist be considered sensitive information?

Reply:
++ I don't think this is something that should be exposed in the client in this manner. Users without access to the Actions framework shouldn't be able to read these configuration values. Can we instead gate these behind a properly authorized API call?

Reply:
I would say that not per se, but the context may reveal info like infrastructure or communication patterns